### ISSUE TITLE
[FEATURE] The ability to specify audio quality

### ISSUE THREAD

**Issue by CHJ85** [nathom/streamrip#475]
### Is the feature request related to a problem. Please describe it.

Hi. There already is the option to specify a max quality. But if you choose to download an entire artist, it'll download all 3 versions of each album (m4a, flac 16 and flac 24 bit). 

### Describe the solution you would like.

It would be nice if I could specify flac 24. Thanks.

**Comment by nathom** [comment]
Do you mean it downloads 3 copies of the same album? It shouldn't do that. What's the command you used?

**Comment by CHJ85** [comment]
Yea. Tidal does that. It's not a big deal though. I just delete the m4a and the 16b flac versions. But it'd be nice to have a flag that'll specify which one I prefer.

**Comment by TheGMan1776** [comment]
> Yea. Tidal does that. It's not a big deal though. I just delete the m4a and the 16b flac versions. But it'd be nice to have a command flag for specifying which one I prefer. But if it's not suppose to do that, I guess it's a bug.

It shouldn't do what you're saying it does, and it didn't do it for me mine worked how it should work which is how you're "requesting" it work. Then I reinstalled windows and now I'm re-setting up StreamRip and I have the issue you're having while using Tidal. I'll post a comment here letting you know if I find a fix.

**Comment by TheGMan1776** [comment]
> > Yea. Tidal does that. It's not a big deal though. I just delete the m4a and the 16b flac versions. But it'd be nice to have a command flag for specifying which one I prefer. But if it's not suppose to do that, I guess it's a bug.
> 
> It shouldn't do what you're saying it does, and it didn't do it for me mine worked how it should work which is how you're "requesting" it work. Then I reinstalled windows and now I'm re-setting up StreamRip and I have the issue you're having while using Tidal. I'll post a comment here letting you know if I find a fix.

So I have 2 PCs. 1 I installed windows like I said above the other I did not. The one I reinstalled windows on and re downloaded everything does what I said in the first comment. The PC I DID NOT install anything on downloads the first version and then seems to TRY to re download the songs but gives the message saying the file already exists and then doesn't re download anything. (I don't think me installing windows is related at all it just so happen that I caught the bug starting to happen because I was reinstalling everything and setting things up)

**Comment by TheGMan1776** [comment]
@CHJ85 Did you have any luck figuring this out?

**Comment by CHJ85** [comment]
@TheGMan1776 Nah. I just download the entire thing and delete the lower qualities afterwords. Not a big deal.

**Comment by nathom** [comment]
@CHJ85 See if v2 on the dev branch fixes the issue.
